service_info/templates/jira/request_for_service.txt

```diff
@@ -0,0 +1,10 @@
+A user has requested the following service be added.
+
+Provider Name:   {{ rfs.provider_name }}
```

Comment:
Let's add a link to the request in the Django admin.  (`absolute_url(self.request_for_service.get_admin_edit_url())`)